django_rest_generator/client.py

```diff
@@ -270,7 +270,9 @@ def _build_from_openapi_schema(self, schema_file: str = None):
 
     @classmethod
     def build_from_openapi_schema(cls, schema_file, token=None):
-        return cls(token=token)._build_from_openapi_schema(schema_file)
```

Comment:
This syntax implies that `build_from_openapi_schema` returns a class instance, but `_build_from_openapi_schema` doesn't return anything.

I would do either
```python3
    @classmethod
    def build_from_openapi_schema(cls, schema_file, token=None):
        cls(token=token)._build_from_openapi_schema(schema_file)
```
if you want nothing returned, or end `_build_from_openapi_schema` with

```python3
return self
```

Reply:
Also, can you add docstrings?